### ISSUE TITLE
Make test selection conditional (dependant on job)

### ISSUE THREAD

**Issue by anastasiyaig** [status-im/desktop-qa-automation#412]
I changed the configuration for new tests to run tests by mark `-m critical` by default (https://github.com/status-im/desktop-qa-automation/pull/394). This results in fact, that now it launches like that in each job, because we have only 1 configuration for new tests hence it applies everywhere. What i need:

1. launch **all** tests  in nightly job
2. launch `-m critical` tests in PRs in status-desktop repository
3. launch configurable selection of tests in PRs in automation repository (like maybe `critical` + `smoke` or whatever)

So basically, i think, I need to add some condition which will recognise job type and possibility to change configurable params for a job based on its type / kind

I need some help with that though

**Comment by yakimant** [comment]
I guess, these are the jobs:
1. https://ci.status.im/job/status-desktop/job/systems/job/linux/job/x86_64/job/tests-e2e-new/
2. https://ci.status.im/job/status-desktop/job/e2e/job/prs/
3. https://ci.status.im/job/status-desktop/job/qa-automation/job/prs/

**Comment by yakimant** [comment]
Can I do `-m all` or smth like that for nightlies?
For 3 I need to `pytest -m critical smoke`?

**Comment by anastasiyaig** [comment]
we did not implement 'all' mark, but pytest command without mark should work the same

**Comment by anastasiyaig** [comment]
for 3 i just was wondering if it could be configured by myself (so in case there is new mark i could add it without bothering you) 
right now -m critical is also fine i just need to learn where it is set

**Comment by yakimant** [comment]
Ok, let's continue in #416.
I need to test it though.

**Comment by yakimant** [comment]
I could not find anything in the
https://ci.status.im/job/status-desktop/job/systems/job/linux/job/x86_64/job/tests-e2e-new/
env vars to differentiate between `nightly` and other runs:
```
TESTRAIL_RUN_NAME=StatusIm-Desktop-231225-035700-55d0a3-nightly
...
BUILD_URL=https://ci.status.im/job/status-desktop/job/systems/job/linux/job/x86_64/job/tests-e2e-new/137/
BUILD_DISPLAY_NAME=#137
BUILD_ID=137
BUILD_TAG=jenkins-status-desktop-systems-linux-x86_64-tests-e2e-new-137
BUILD_NUMBER=137
BUILD_SOURCE=status-desktop/systems/linux/x86_64/package
```

We need to revise the condition, when other types of runs will appear in the future
FYI @anastasiyaig, @jakubgs

**Comment by jakubgs** [comment]
The job name is the difference, it only runs nightlies. At least for now...
